.eslintrc.json

```diff
@@ -1,7 +1,7 @@
 {
     "env": {
         "commonjs": true,
-        "es6": false,
+        "es6": true,
```

Comment:
Changed this as we are running on node version 8.16.0 which supports ES6 features.